wp-content/themes/goonj-crm/functions.php

```diff
@@ -562,6 +583,28 @@ function goonj_redirect_after_individual_creation() {
 				);
 				break;
 			}
+			case 'dropping-center-contribution':
+				if ( ! $source ) {
+					\Civi::log()->warning('Source is missing for material contribution flow', ['individualId' => $_GET['individualId']]);
+					return;
+				}
+				// If the individual was created while in the process of material contribution,
+				// then we need to find out from WHERE was she trying to contribute.
+	
+				// First, we check if the source of Individual is Colllection Camp (or Dropping Center).
+				$collectionCamp = \Civi\Api4\EckEntity::get( 'Collection_Camp', false )
+					->addWhere( 'title', '=', $source )
+					->setLimit( 1 )
```

Comment:
Why we are setting limit here, i don't think it's needed can you please test it?

Reply:
because here we want only one entry.

Reply:
As we are using the `->execute()->first();` it always extract one entry so why we need the `->setLimit( 1 )`

Reply:
oh , correct , I have remove `->setLimit( 1 )`